AreaCalculator

```diff
@@ -1,20 +1,19 @@
 public class AreaCalculator {
 
+
     public static double area (double radius) {
         if (radius < 0) {
             return -1.0;
         } else {
-            double circleArea = radius*radius*Math.PI;
```

Comment:
The else is completely redundant here. 
It could be just
```
   if (radius < 0) {
      return -1.0; 
   } 
   return  radius * radius * Math.PI;
```